src/pages/User.js

```diff
@@ -0,0 +1,44 @@
+import logo from '../CodeKingTrans.png';
+import Navbar from '../components/Navbar.js';
+import Footer from '../components/Footer.js';
+import List from '../components/List.js';
+import useFetch from '../hooks/useFetch';
+import '../App.css';
+
+function UserCard({ user }){
+    return (
+        <div className='userCard'>
+            <img alt='profile logo' src='https://cdn-icons-png.flaticon.com/512/12225/12225773.png'></img>
+            <h2>{user.firstName + ' ' + user.lastName}</h2>
+        </div>
+    );
+}
+
+
+function User() {
+    const { data: users, loading, error } = useFetch("https://disc-assignment-5-users-api.onrender.com/api/users");
+    
+
+
+    return (
+        <div className="App">
+            <Navbar />
+            <div className='backdrop'>
+                <img alt='Code King Logo' src={logo}></img>
+                <div className="userList">
+                    <div className='userListTitle'>
+                        <h1>Friends:</h1>
+                    </div>
+                    {loading && <p>Loading...</p>}
+                    {error && <p>Error: {error.message}</p>}
+                    {users && (
+                        <List items={users} renderItem={(user) => <UserCard key={user.id} user={user} />} />
+                    )}
+                </div>
+            </div>
+            <Footer />
+        </div>
+    );
```

Comment:
To make your code more readable, I encourage you to make more components that hide away implementation details from top-level components (components at the top of the tree).

Think about it this way. Someone is opening the `User.js` file because they want to know what's on the users page. Instead of telling them in great detail every html element on the page and what those details look like, you could instead provide a short summary of what's on the page. for example, you can say the users page is just a navbar, a logo, a list of users, and a footer.
Your return statement would then look like
```js
return (
  <div>
    <Navbar/>
    <Logo />
    <UsersList />
    <Footer />
  </div>
```

Then, if someone wants more detail about any specific part of the page, for example they want more detail about what is a "UsersList", they can go to `UsersList.js` file which defines the `UsersList` component. 
In this way, your `User.js` component really just acts like a table of contents instead of a detailed description of everything on the page. This is useful when a component can be broken into multiple subparts or subsections that each have a distinct purpose, or when components get very big and long.